src/crimson/os/seastore/seastore.cc

```diff
@@ -1923,7 +1936,8 @@ SeaStore::Shard::_do_transaction_step(
         last = i.decode_string();
         return _omap_rmkeyrange(
 	  ctx, onodes[op->oid],
-	  std::move(first), std::move(last));
+	  std::move(first), std::move(last),
+	  onodes[op->oid]->get_layout().log_root);
       }
       case Transaction::OP_OMAP_CLEAR:
```

Comment:
Not sure, if OSD was calling `OP_OMAP_CLEAR` to clear the omap (including the logs), SeaStore/FuturizedStore should expose a symmetric `OP_LOG_CLEAR` operation to do so after splitting out logs, right?

CC @Matan-B

Reply:
OK, it can be introduced when necessary.

Reply:
follow up: https://tracker.ceph.com/issues/69239